redbot/cogs/alias/alias_entry.py

```diff
@@ -90,6 +90,32 @@ def __init__(self, config: Config, cache_enabled: bool = True):
         self._loaded = False
         self._aliases: Dict[Optional[int], Dict[str, AliasEntry]] = {None: {}}
 
+    async def anonymize_aliases(self, user_id: int):
+
+        async with self.config.entries() as global_aliases:
+            for a in global_aliases:
+                if a.get("creator", 0) == user_id:
+                    a["creator"] = 0xDE1
+                    if self._cache_enabled:
+                        self._aliases[None][a["name"]] = AliasEntry.from_json(a)
+
+        all_guilds = await self.config.all_guilds()
+        async for guild_id, guild_data in AsyncIter(all_guilds.items(), steps=100):
+            for a in guild_data["entries"]:
+                if a.get("creator", 0) == user_id:
+                    break
+            else:
+                continue
+            # basically, don't build a context manager wihout a need.
+            async with self.config.guild_from_id(guild_id).entries as entry_list:
+                for a in entry_list:
+                    if a.get("creator", 0) == user_id:
+                        # fmt: off
+                        a["creator"] = 0xde1
+                        # fmt: on
```

Comment:
Seems a bit weird to do this in this place but no other.
```suggestion
                        a["creator"] = 0xde1
```